python/py-makelive/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-makelive
 version             0.5.0
-revision            0
+revision            1
```

Comment:
if you're fixing a build error and/or adding a build dependency there is no reason to increase the `revision`